site/installation/README.md

```diff
@@ -1,5 +1,7 @@
 Users can get kpt in a variety of ways:
 
+**Note:** Please follow the [migration guide] to make your existing kpt packages compatible
```

Comment:
To future proof this, reword:

If you're currently using version v0.39, follow the ... to migrate.